### ISSUE TITLE
Primus message store ERR_BLOCKED_BY_CLIENT

### ISSUE THREAD

**Issue by PiiXiieeS** [stephenou/fruitionsite#7]
Any idea of how to get rid off the regular message emitted by the msgstore.www.notion.so.
Also I notice a very weird behaviour; while websockets seems to work well in iOS iPhone device, seeing pages being updated almost in realtime, it is not the case in desktop client, both Safari and Chrome.

```
GET https://msgstore.www.notion.so/primus/?sessionId=0dac675a-94e0-5cd6-766f-b6a037fecb12&_primuscb=N7_cHfF&EIO=3&transport=polling&t=N6_cHfF&b62=1 net::ERR_BLOCKED_BY_CLIENT
```

**Comment by stephenou** [comment]
Yeah, I know about this but haven't had time to investigate how to make WebSockets works for us.

The gist of the problem is WebSockets (which enables push) fails on desktop, which is why it falls back to polling. The request you're seeing is periodic polling.

**Comment by SuperChenSSS** [comment]
Hi @stephenou  I was also encountering this problem. Looks like the website has already failed with HTTP400 on this domain. May I know if any fix about that?

BTW, the error message is: Session ID unknown for me. I sent a GET to that URL and also returned the same error, the screenshot is as the below part:
<img width="967" alt="image" src="https://user-images.githubusercontent.com/17025439/161596111-76443969-b576-4edf-9cb1-3b59fceedfa0.png">

**Comment by dziter** [comment]
Hi, may I ask you how did you fix it?
cc @stephenou @PiiXiieeS @SuperChenSSS
